src/drivers/ports/esp32/can_port.h

```diff
@@ -31,23 +31,23 @@
 #ifndef __DRIVERS_CAN_PORT_H__
 #define __DRIVERS_CAN_PORT_H__
 
-/* #define CAN_PORT_SPEED_1000KBPS (ESP32_CAN_BTIM0_BRP(0x6)               \ */
-/*                                  | ESP32_CAN_BTIM0_SJW(3)               \ */
-/*                                  | ((ESP32_CAN_BTIM1_TSEG1(2)           \ */
-/*                                      | ESP32_CAN_BTIM1_TSEG2(3)         \ */
-/*                                      | ESP32_CAN_BTIM1_SAM) << 8)) */
+#define CAN_PORT_SPEED_1000KBPS (ESP32_CAN_BTIM0_BRP(0x4)         \
+                                 | ESP32_CAN_BTIM0_SJW(0x1)       \
+                                 | ((ESP32_CAN_BTIM1_TSEG1(0x4)   \
+                                 | ESP32_CAN_BTIM1_TSEG2(0x1)     \
+                                 | ESP32_CAN_BTIM1_SAM) << 8))
+                                   
+#define CAN_PORT_SPEED_500KBPS (ESP32_CAN_BTIM0_BRP(0x4)          \
+                                | ESP32_CAN_BTIM0_SJW(0x1)        \
+                                | ((ESP32_CAN_BTIM1_TSEG1(0xC)    \
```

Comment:
Change 0xC to 0xc in both places, as that is the undocumented coding convension =)